english/marshmallow.md

```diff
@@ -6,8 +6,7 @@ and search for your [destiny](destiny/destiny.md).
 
 Before we begin, do you need a [warm-up?](warm-up/warm-up.md)
 
-You are are man (or woman) for all seasons but need help in choosing
-your travel destinations.. Of course we are happy to help you with these
+Are you a person for all seasons but need help in choosing a travel destinations.. Of course we are happy to help you with these
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.